src/components/timeline/ActiveTabTimeline.css

```diff
@@ -28,9 +28,78 @@
 }
 
 .timelineResourcesHeader::before {
-  position: absolute;
   top: calc(50% - 5px);
   left: 8px;
+}
+
+.timelineResourceTracks {
+  position: relative;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+/* Active tab timeline resource tracks */
+.timelineTrackResource:first-child {
+  border-top: none;
+}
+
+.timelineTrackResourceRow {
+  position: relative;
+  flex-direction: column;
+}
+
+.timelineTrackResourceRow .timelineTrackThreadMarkers {
+  width: 100%;
+}
+
+.timelineTrackResourceRow .threadActivityGraph,
+.timelineTrackResourceRow .timelineMarkers {
+  transition: height 0.2s;
+}
+
+.timelineTrackResourceRow:not(.opened) .timelineMarkers {
+  height: 3px;
+}
+
+.timelineTrackResourceRow.opened
+  .timelineMarkers:not(.timelineMarkersGeckoMain)
```

Comment:
This selector is pretty surprising. I would suggest either:

A: Change to a simpler class-based selector.
B: Document what is going on here.